Client/Ecosia/EngagementServices/EngagementService.swift

```diff
@@ -0,0 +1,32 @@
+import Foundation
+import Core
+
+class ClientEngagementService {
+    
+    private init() {}
+    
+    static let shared = ClientEngagementService()
+    private let service  = EngagementService(provider: Braze())
+    
+    func initialize(parameters: [String: Any]) {
+        do {
+            try service.initialize(parameters: parameters)
+        } catch {
+            debugPrint(error)
```

Comment:
Do we plan on doing error handling in the future? Or just like Singular we don't care since it is not user facing?

Reply:
Thanks for asking!
At this moment, we are treating it as Singular. There is no UX planned to notify the user in case of a wrong initialization of the Braze (or any other) engagement service.